skll/data/readers.py

```diff
@@ -936,7 +940,7 @@ def __init__(self, path_or_list, **kwargs):
 
     @staticmethod
     def split_with_quotes(
-        string: str, delimiter=" ", quote_char="'", escape_char="\\"
+        string: str, delimiter: str = " ", quote_char: str = "'", escape_char: str = "\\"
```

Comment:
```suggestion
        string: str, delimiter: str = " ",
        quote_char: str = "'",
        escape_char: str = "\\"
```
seems more readable to me

Reply:
Me too but I don't want to override `black` unless absolutely necessary.